.changeset/serious-pigs-watch.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-explore': patch
+'@backstage/plugin-search-react': patch
+---
+
+`ListItem` wrapper component moved to `SearchResultListItemExtension` for all `*SearchResultListItems`. This is to make sure the list only contains list elements.
+
+Note: If you have implemented a custom result list item, you can remove the list item wrapper to avoid duplicated `<li>` elements.
```

Comment:
Is it breaking 🤔 ?

Reply:
Nope, but definitely not recommended! Maybe I can make that more clear with something like

> Note: If you have implemented a custom result list item, we recommend you to remove the list item wrapper to avoid nested `<li>` elements.

Reply:
Looks great to me, thanks for explaining 🙌🏻